projects/optic/src/commands/diff/diff.ts

```diff
@@ -215,6 +215,14 @@ const runDiff = async (
     console.log(log);
   }
 
+  const diffResults = {
+    checks: {
+      total: specResults.results.length,
+      passed: specResults.results.filter((check) => check.passed).length,
+      failed: specResults.results.filter((check) => !check.passed).length,
+    },
+  };
```

Comment:
do we need to handle exemptions (i think here it's inline exemptions)

Reply:
good note. I've updated the failed predicate to check if exempted is set